scripts/deployment/phase2-assets/collaterals/deploy_morpho_aavev2_plugin.ts

```diff
@@ -181,7 +181,7 @@ async function main() {
         maxTradeVolume: fp('1e6'), // $1m,
         oracleTimeout: '86400', // 24 hr
         targetName: ethers.utils.formatBytes32String('BTC'),
-        defaultThreshold: fp('0.01').add(combinedBTCWBTCError), // ~3.5%
+        defaultThreshold: fp('0.01').add(wbtcOracleError), // 3%
```

Comment:
same thing here, I'm seeing in `MorphoNonFiatCollateral` that the two prices are multiplied together

Reply:
Same thing as for `cbETH`.. `pegPrice` uses a single oracle.